tests/e2e/instrumentation-java/00-install-collector.yaml

```diff
@@ -21,3 +21,6 @@ spec:
           receivers: [otlp]
           processors: []
           exporters: [logging]
+      telemetry:
```

Comment:
I've put back that configuration because we have removed it on the following PR: https://github.com/open-telemetry/opentelemetry-operator/pull/680. 
 I agree that we should explore the test assertion and remove them from the auto-instrumentation.